src/bakery/Bakery.cpp

```diff
@@ -320,3 +320,11 @@ void Bakery::solveThirdPhase(bool useKnapsack, bool optimize) {
 const vector<Van> &Bakery::getVans() const {
     return vans;
 }
+
+void Bakery::calculateSCC() {
+    this->graph.calculateSccTarjan(this->startingVertex);
+}
+
+void Bakery::displayAllScc() {
+    this->graph.displaySccTarjan();
+}
```

Comment:
These functions seem unnecessary. I would just call the SCC function from graph and apply the right algorithms there